tests/fXmlRpc/Integration/NodeIntegrationTest.php

```diff
@@ -39,23 +39,4 @@ class NodeIntegrationTest extends AbstractIntegrationTest
     protected static $command = 'exec node server.js';
 
     protected static $restartServerInterval = 200;
-
-    /**
-     * @dataProvider getClients
-     */
-    public function testServerNotReachableViaTcpIp(ClientInterface $client)
```

Comment:
Because it in fact tests a Transport behaviour. But that is one that I am not really sure about. It looks like it is a client behaviour, but it is not. The exception is thrown in a transport. Since transports are being replaced, I rely on the default behaviour of the adapter package (which is already tested).